.github/workflows/ci.yaml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
-
+    
```

Comment:
Extraneous tab that shouldn't exit (I think)

Reply:
I have no idea now it ended up here 😆